charts/newrelic-infrastructure-v3/templates/NOTES.txt

```diff
@@ -4,13 +4,112 @@ Your deployment of the New Relic Infrastructure agent is complete if you have no
+-> WARNING LEGACY CONFIG <-
+You have specified the legacy "tolerations" options into values, however the new version of the chart
+supports different tolerations per component, therefore the value specified has been ignored.
+
+Please use ksm.tolerations, controlPlane.tolerations and kubelet.tolerations.
+{{- end }}
+
+{{- if .Values.image }}
+
+-> WARNING LEGACY CONFIG <-
+You have specified into values one of the legacy options "image.*".
+The following values are no longer supported and are currently ignored.
+ - image.repository
+ - image.tag
+ - image.pullPolicy
+ - image.pullSecrets
+
+Notice that the 3.x version of the integration uses 3 different images.
+Please set:
+ - images.forwarder.* to configure the image in charge of sending data to newrelic backend
+ - images.agent.* to configure the image bundling the agent and onHost integration
+ - images.integration.* to configure the image in charge of scraping k8s data
+{{- end }}
+
+{{- if (or .Values.enableWindows .Values.windowsOsList .Values.windowsSecurityContext .Values.windowsNodeSelector)}}
+
+-> WARNING LEGACY CONFIG <-
+Windows is currently supported by 2.x charts only and therefore "windowsOsList" "windowsSecurityContext" "windowsNodeSelector"
+are no longer supported.
+
+{{- end }}
+
+{{- if ( or .Values.controllerManagerEndpointUrl  .Values.schedulerEndpointUrl .Values.etcdEndpointUrl .Values.apiServerEndpointUrl )}}
+
+-> WARNING LEGACY CONFIG <-
+Values "controllerManagerEndpointUrl", "etcdEndpointUrl", "apiServerEndpointUrl", "schedulerEndpointUrl" are no longer
+supported, please specify them in the config file. You can add them in the autodiscovery section. Example for etcdEndpointUrl:
+
+#  autodiscover:
+#    - selector: "tier=control-plane,component=etcd"
+#      namespace: kube-system
+#      matchNode: true
+#      endpoints:
+#        - url: https://localhost:4001
+#          insecureSkipVerify: true
+#          auth:
+#            type: bearer
+
+{{- end }}
+
+{{- if ( or .Values.etcdTlsSecretName .Values.etcdTlsSecretNamespace )}}
+
+-> WARNING LEGACY CONFIG <-
+Values "etcdTlsSecretName" and "etcdTlsSecretNamespace" are no longer supported, please specify them in the config
+file. Example:
+# - endpoints:
+#     - url: https://localhost:9979
+#       insecureSkipVerify: true
+#       auth:
+#         type: mTLS
+#         mtls:
+#           secretName: secret-name
+#           secretNamespace: secret-namespace
+{{- end }}
+
+{{- if .Values.logFile }}
+
+-> WARNING LEGACY CONFIG <-
+The "logFile" value is no longer supported, please specify the option "log_file" directly in "common.agentConfig".
+
+Notice that you will need as well to set securityContext.readOnlyRootFilesystem to false or add extraVolumes to allow
+the container to write to such location
+{{- end }}
+
+{{- if .Values.apiServerSecurePort }}
 
+-> WARNING LEGACY CONFIG <-
+The "apiServerSecurePort" value is no longer supported, please specify it in the section "apiServer.autodiscover[].endpoints".
```

Comment:
`{{- fail "Please migrate your control plane config to the nri-kubernetes v3 format " }}`